target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -82,6 +82,9 @@ ipq40xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0u@eth0" "3:lan" "4:lan"
 		;;
+	devolo,dlan2-2400-ac)
+		ucidef_set_interface "lan" ifname "eth0 eth1 eth2" protocol "dhcp"
```

Comment:
I'm not a fan of this. I think, by default, these devices should act as a user would expect it from a "standard" OpenWrt device.

Reply:
The intent was to more closely replicate how the devices are used with the retail firmware, where they have no router functionality. I've updated it to use the default set_interface_lan.

Reply:
Thanks for changing, note that I understand your point. We had a similar discussion elsewhere recently, and this is effectively more a matter of policy than an objective "best" way to do it; but in this case I'd actually prefer to have a standard OpenWrt way how the device should be accessible after first installation.